shared/util/emoji.tsx

```diff
@@ -69,7 +72,11 @@ export function RPCToEmojiData(emoji: RPCChatTypes.Emoji, category?: string): Em
     userEmojiRenderStock:
       emoji.source.typ === RPCChatTypes.EmojiLoadSourceTyp.str ? emoji.source.str : undefined,
     userEmojiRenderUrl:
-      emoji.source.typ === RPCChatTypes.EmojiLoadSourceTyp.str ? undefined : emoji.source.httpsrv,
+      emoji.source.typ === RPCChatTypes.EmojiLoadSourceTyp.str
+        ? undefined
+        : noAnim && emoji.noAnimSource.typ === RPCChatTypes.EmojiLoadSourceTyp.httpsrv
+        ? emoji.noAnimSource.httpsrv
+        : emoji.source.httpsrv,
```

Comment:
Seems better to plumb both URLs here into `EmojiData`, and pick one in `emojiDataToRenderableEmoji` so it's consistent with how `UserReacji` works

Reply:
It is consistent with the RPC part of the whole thing.

Reply:
Not sure I understand. In my mind EmojiData just describe the emoji, and RenderableEmoji controls how an emoji is rendered. It only matters whether to animate when the emoji is rendered, so the split should happen when we create RenderedEmoji. The way UserReacji works in this PR matches this concept, but EmojiData does not.